### ISSUE TITLE
Detection of Yolo

### ISSUE THREAD

**Issue by ZHUXUHAN** [Maelic/SGG-Benchmark#49]
dear author, I saw in your paper that you did not train for object classification. So why did your mAP detection accuracy increase, because theoretically, the performance of detection should remain unchanged. I hope to receive your response

**Comment by Maelic** [comment]
If you are talking about the training of the YoloV8 model, it was trained for object detection (bounding box regression + object classification) beforehand and fully frozen during relation prediction training. YOLOV8 is by default way better than Faster-RCNN for object detection, that's why this strategy achieves higher overall mAP.

**Comment by ZHUXUHAN** [comment]
> If you are talking about the training of the YoloV8 model, it was trained for object detection (bounding box regression + object classification) beforehand and fully frozen during relation prediction training. YOLOV8 is by default way better than Faster-RCNN for object detection, that's why this strategy achieves higher overall mAP.

The accuracy of your pre-trained YOLOv8 model is around 26, but it increases to 31 in the SGG task without training object classification or detection. Why does it increase in performance in this case?

**Comment by Bobbypower** [comment]
> The accuracy of your pre-trained YOLOv8 model is around 26, but it increases to 31 in the SGG task without training object classification or detection. Why does it increase in performance in this case?
> 
> Because the Dynamic Candidate Selection method? Hope for your answers.

In my dataset, the performance improved by approximately 4 points,

**Comment by ZHUXUHAN** [comment]
> > The accuracy of your pre-trained YOLOv8 model is around 26, but it increases to 31 in the SGG task without training object classification or detection. Why does it increase in performance in this case?
> > Because the Dynamic Candidate Selection method? Hope for your answers.
> 
> In my dataset, the performance improved by approximately 4 points,

Yes, I see, I want to know why, because the object detection are not trained.

**Comment by Maelic** [comment]
> > If you are talking about the training of the YoloV8 model, it was trained for object detection (bounding box regression + object classification) beforehand and fully frozen during relation prediction training. YOLOV8 is by default way better than Faster-RCNN for object detection, that's why this strategy achieves higher overall mAP.
> 
> The accuracy of your pre-trained YOLOv8 model is around 26, but it increases to 31 in the SGG task without training object classification or detection. Why does it increase in performance in this case?
> 
> Because the Dynamic Candidate Selection method? Hope for your answers.

Hi,
First of all, I have not yet released the Dynamic Candidate Selection code (I will do it soon).
The accuracy of YOLOV8 should stay the same before and after training because the model is completely frozen during the relation training stage. Are you sure you are evaluating the YOLOV8 model on the test set and not the validation set?

**Comment by ZHUXUHAN** [comment]
> > > If you are talking about the training of the YoloV8 model, it was trained for object detection (bounding box regression + object classification) beforehand and fully frozen during relation prediction training. YOLOV8 is by default way better than Faster-RCNN for object detection, that's why this strategy achieves higher overall mAP.
> > 
> > 
> > The accuracy of your pre-trained YOLOv8 model is around 26, but it increases to 31 in the SGG task without training object classification or detection. Why does it increase in performance in this case?
> > Because the Dynamic Candidate Selection method? Hope for your answers.
> 
> Hi, First of all, I have not yet released the Dynamic Candidate Selection code (I will do it soon). The accuracy of YOLOV8 should stay the same before and after training because the model is completely frozen during the relation training stage. Are you sure you are evaluating the YOLOV8 model on the test set and not the validation set?

Thanks for your answers, I understand it, hope for your new uploaded codes.
